src/db/seeders/20210118210721-owner.js

```diff
@@ -0,0 +1,116 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert("Owners", [
```

Comment:
if you are seeding a lot of items, it might be worth putting this data in its own .js file and mapping over it in this file. It's nice to have big arrays or json objects in their own file, in case it needs to be used in other places in the backend. Here is an example of what I did on scgts to seed roles. 
![Screen Shot 2021-01-18 at 5 48 40 PM](https://user-images.githubusercontent.com/31671880/104971832-99d8b180-59b5-11eb-93f0-1cea34095a11.png)
![Screen Shot 2021-01-18 at 5 48 49 PM (2)](https://user-images.githubusercontent.com/31671880/104971841-9e04cf00-59b5-11eb-819e-44a99c4a8384.png)